extensions/data-workspace/src/main.ts

```diff
@@ -17,9 +17,9 @@ import { ProjectDashboard } from './dialogs/projectDashboard';
 export async function activate(context: vscode.ExtensionContext): Promise<IExtension> {
 	const workspaceService = new WorkspaceService(context);
 	await workspaceService.loadTempProjects();
-	await workspaceService.checkForProjectsNotAddedToWorkspace();
-	context.subscriptions.push(vscode.workspace.onDidChangeWorkspaceFolders(async () => {
-		await workspaceService.checkForProjectsNotAddedToWorkspace();
+	workspaceService.checkForProjectsNotAddedToWorkspace();
```

Comment:
can you add a comment here as to why they should not be added just for future

Reply:
and while you're at it add a `.catch(e => console.log(e))` to properly handle the un-awaited promise (and make it clearer that it's done on purpose)